content/workshops/cascading-layouts.md

```diff
@@ -67,11 +86,16 @@ summary: |
 {% import 'utility.macros.njk' as utility %}
 {% import 'contact.macros.njk' as contact %}
 {% import 'embed.macros.njk' as embed %}
+{% import "quotes.macros.njk" as quotes %}
 
 <div id="workshop-video">
   {{ embed.figure([{ youtube: '8k6m87woEpQ' }]) }}
 </div>
 
+<div id="workshop-video">
```

Comment:
```suggestion
<div id="workshop-video-2">
```

This needs to be unique, and then you'll be able to use this as the direct link to this video.